src/llmcompressor/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -294,17 +293,10 @@ def update_and_save_recipe(model_stub: str, save_directory: str):
         existing recipe
     :param save_directory: path to save combined existing recipe and current recipe
     """
-    recipes_to_save = []
-    existing_recipe = infer_recipe_from_model_path(model_stub)
-    if existing_recipe is not None:
-        recipes_to_save.append(existing_recipe)
 
-    new_recipe = active_session().lifecycle.recipe
-    if new_recipe is not None:
-        recipes_to_save.append(new_recipe)
+    existing_recipe = infer_recipe_from_model_path(model_stub)
 
-    recipe = Recipe.simplify_combine_recipes(recipes_to_save)
+    recipe = active_session().lifecycle.recipe
 
-    # save recipe
     recipe_path = os.path.join(save_directory, RECIPE_FILE_NAME)
-    recipe.yaml(recipe_path)
+    recipe.yaml(file_path=recipe_path, existing_recipe_path=existing_recipe)
```

Comment:
I thought we were going to save yaml files sequentially, rather than merging them?